tests/acceptance/features/bootstrap/WebDavPropertiesContext.php

```diff
@@ -229,18 +172,16 @@ public function userHasSetFollowingPropertiesUsingProppatch(string $username, st
 		$username = $this->featureContext->getActualUsername($username);
 		$this->featureContext->verifyTableNodeColumns($propertiesTable, ['propertyName', 'propertyValue']);
 		$properties = $propertiesTable->getColumnsHash();
-		$this->featureContext->setResponse(
-			WebDavHelper::proppatchWithMultipleProps(
-				$this->featureContext->getBaseUrl(),
-				$username,
-				$this->featureContext->getPasswordForUser($username),
-				$path,
-				$properties,
-				$this->featureContext->getStepLineRef(),
-				$this->featureContext->getDavPathVersion()
-			)
+		$response = WebDavHelper::proppatchWithMultipleProps(
+			$this->featureContext->getBaseUrl(),
+			$username,
+			$this->featureContext->getPasswordForUser($username),
+			$path,
+			$properties,
+			$this->featureContext->getStepLineRef(),
+			$this->featureContext->getDavPathVersion()
 		);
-		$this->featureContext->theHTTPStatusCodeShouldBeSuccess();
+		$this->featureContext->theHTTPStatusCodeShouldBeBetween(200, 299, $response);
```

Comment:
I guess we can do something like this right? Propatch should be multi status so maybe let's be specific here
```suggestion
		$this->featureContext->theHTTPStatusCodeShouldBe(207,response);
```